code/unclassified/src/min_max_of_an_array_using_min_comparisons/Added_comments_Maximum_and_Minimum_of_an_array_using_minimum_comparisons_(Tournament Method).cpp

```diff
@@ -26,16 +28,19 @@ struct minMax tournamentMethod(std::vector<int> a, int beg, int end) {
         }
         return min_max;
     }
-
-    int mid = (beg + end) / 2;     //If there are more than 2 elements
+    //If there are more than 2 elements
+    int mid = (beg + end) / 2;     
```

Comment:
@mehakmalhotra1 it is the wrong way to find mid element this may cause an overflow. Use `mid = beg + (end - beg) / 2`